graspy/inference/latent_distribution_test.py

```diff
@@ -14,121 +14,176 @@
 
 import numpy as np
 
-from ..embed import AdjacencySpectralEmbed, select_dimension
+from ..embed import select_dimension, AdjacencySpectralEmbed
 from ..utils import import_graph, is_symmetric
 from .base import BaseInference
+from sklearn.metrics import pairwise_distances
+from sklearn.metrics.pairwise import PAIRED_DISTANCES
+from hyppo.ksample import KSample
+
+_VALID_METRICS = PAIRED_DISTANCES.keys()
+_VALID_TESTS = ["cca", "dcorr", "hhg", "rv", "hsic", "mgc"]
 
 
 class LatentDistributionTest(BaseInference):
     """
-    Two-sample hypothesis test for the problem of determining whether two random 
+    Two-sample hypothesis test for the problem of determining whether two random
     dot product graphs have the same distributions of latent positions.
-    
+
     This test can operate on two graphs where there is no known matching between
-    the vertices of the two graphs, or even when the number of vertices is different. 
```

Comment:
@alyakin314 just a note, i am removing this for now until we get your fix

Reply:
agree.